utils/build-script

```diff
@@ -758,16 +758,17 @@ details of the setups of other systems or automated environments.""")
              "implementation for Android builds",
         metavar="PATH")
     android_group.add_argument(
-        "--android-ndk-version",
-        help="A version of the NDK to use when building for Android. "
+        "--android-api-level",
+        help="The Android API level to target when building for Android. "
              "Currently only 21 or above is supported",
         default="21")
     android_group.add_argument(
         "--android-ndk-toolchain-version",
         help="A version of the toolchain to use when building for Android. "
-             "Use 4.8 for 32-bit builds, 4.9 for 64-bit builds",
+             "Use 4.8 for 32-bit builds, 4.9 for 64-bit builds. "
+             "%(default)s by default",
         choices=["4.8", "4.9"],
-        default="4.8")
```

Comment:
Right, r11c removes 4.8, so this line that sets 4.8 as the default had also been removed. I'm not sure what your concern is. Let me know if I'm missing something!